### DISCUSSION TITLE
Accidentally locked myself out from my NAS

### DISCUSSION THREAD

**Discussion by br0kenpixel** [iterate-ch/cyberduck#16518] [Q&A] [answered]
Hello.
I was trying to set up an SFTP bookmark for my NAS. When it asked me to accept the fingerprint or whatever I checked "Always" and accidentally clicked "Decline" and now I'm locked out. How can I fix this? Why is there nothing in the settings to undo stuff like this?

**Comment by dkocher** [comment, marked as answer]
Delete the corresponding entry from `.ssh/known_hosts`.

**Reply by br0kenpixel** [reply]
That worked, thank you!
